experimental/dds/tree2/src/shared-tree/sharedTreeView.ts

```diff
@@ -335,6 +356,9 @@ export class SharedTreeView implements ISharedTreeView {
 				this._events.emit("afterBatch");
 			}
 		});
+		branch.on("revertible", (type) => {
+			this._events.emit("revertible", type, this);
```

Comment:
Is it possible to make SharedTreeView implement a more strongly typed revertible event? A subscriber to the revertible event who does use the `target` argument of the event callback will get an `ISharedTreeView` but not a `SharedTreeView`, even though it is definitely a `SharedTreeView`.

This is probably pretty inconsequential, but it is misleading/weird at worse.

Reply:
I'm going to remove the target actually. It was included because that was the pattern for the undo redo stack manager but anyone using these events should already have access to the view they're being emitted from.